requirements-dev.txt

```diff
@@ -26,3 +26,12 @@ docutils==0.17.1
 
 # Needed by test/test_sockets.py
 websockify==0.10.0
+
+# Specify dependencies compatible with sphinx==2.4.4
+# See https://github.com/emscripten-core/emscripten/issues/21590
+sphinxcontrib-applehelp<=1.0.4
+sphinxcontrib-devhelp<=1.0.2
+sphinxcontrib-htmlhelp<=2.0.0
+sphinxcontrib-serializinghtml<=1.1.5
+sphinxcontrib-qthelp<=1.0.3
+alabaster<=0.7.12
```

Comment:
Can you put these up by the `jinja2` dep, which exists for the same reason (i.e. sphinx compat).

Strange the sphinx itself doesn't specify these ..

Reply:
@sbc100 Done.